app/javascript/src/js/analytics.js

```diff
@@ -81,27 +65,52 @@ $(() => {
   });
 
   // Fire an event whenever someone clicks the normal sign up button (e.g. not step 2 of the github flow)
-  $('[value="Sign up"]').on('click', (e) => {
+  document.querySelector('input[value="Sign up"]')?.addEventListener('click', () => {
     _gaq.push(['_trackEvent', 'signup_buttons', 'sign_up', 'main_signup_button', 1]);
   });
 
   // Fire an event whenever an ad is clicked
-  $('.ad').on('click', (e) => {
-    adId = $(e.target).parents('.ad').data('ad-id');
-    _gaq.push(['_trackEvent', 'ad', 'click', adId, 1]);
-  });
-
-  $('[title="Mark lesson complete"]').on('click', (e) => {
-    _gaq.push(['_trackEvent', 'lesson_completion', 'mark_completed', window.location.href, 1]);
+  document.querySelectorAll('.ad').forEach((ad) => {
```

Comment:
I think this one might be broken. I removed the ad class when moving the lesson page over to tailwind 🙈 

Could we use a href attribute selector like the one we used with discord please? 
`document.querySelectorAll('[href^="https://www.thinkful.com/?utm_source=odin"]')`